packages/replay/.eslintignore

```diff
@@ -4,3 +4,4 @@ demo/build/
 # TODO: Check if we can re-introduce linting in demo
 demo
 metrics
+src/types
```

Comment:
m: Why are we ignoring this?

Reply:
there used to be a problem with eslint ignore comment in the rrweb types file... let me try again if that is still happening!

Reply:
So I checked again, and for whatever reason it is ignoring `/* eslint-disable */` at the top of the `types/rrweb.ts` file. I will narrow this down to this file and add a comment why this is ignored!